qhana_plugin_registry/util/config/__init__.py

```diff
@@ -49,6 +49,10 @@ class ProductionConfig(SQLAchemyProductionConfig, SmorestProductionConfig):
     INITIAL_PLUGIN_SEEDS = []  # a list of URLs
     PRECONFIGURED_SERVICES = []  # a list of dicts with service info
 
+    # rewrite rules for URLs e.g. to map localhost to docker container name
+    URL_MAP_FROM_LOCALHOST = {}
```

Comment:
add comments, that these options will be processed into lists of (key, value) tuples to avoid confusion